app/flows/check_uk_visa_flow/outcomes/outcome_work_y_academic.erb

```diff
@@ -79,5 +79,5 @@
 
   <%= render partial: 'country_in_youth_mobility_scheme', locals: { calculator: calculator } %>
 
-  <%= render partial: 'uk_ancestry_visa' %>
+  <%= render partial: 'uk_ancestry_visa', locals: {calculator: calculator } %>
```

Comment:
Nitpick: the front curly bracket needs a trailing space. Think the same is true across the other outcomes.

Reply:
Good spot :+1:, thanks, now fixed.